### PR TITLE
Add API endpoint

### PR DESCRIPTION
**Pull request by mrkamin**
- [ ] In this PR below tasks are done

- Add an API endpoint that will list all the reservations for a given user.

Capture

### CODE REVIEW

**Review by Iqbal-Elham — COMMENTED** [review]
Hi @mrkamin , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️

Cheers and Happy coding!👏👏👏

**Review by Iqbal-Elham — APPROVED** [review]

File: src/components/home/Home.js [hunk 1]
```diff
@@ -1,10 +1,46 @@
...
+          <img className="photo" src={doctor.photo} alt={doctor.name} />
+          <h2>{doctor.name}</h2>
+          <p>{doctor.about}</p>
+          <p>
+            Buy one hour of time with only $
+            {doctor.price_hour}
+          </p>
+        </div>
+      ))}
+    </div>
```
Comment (Muhammad0602) [inline src/components/home/Home.js]: You are displaying the time as a whole string, but that is not a good user experience. You should remove the unnecessary part of that string. You could also change the number of the month to its name, making it easier for the user to understand.
You could try something like 23 June, 2023.

File: src/components/reservation/Reservations.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          <h1>{reservation.doctor}</h1>
+          <h3>
+            {reservation.city}
+            Kabul
+          </h3>
+          <p>{reservation.appointment_time}</p>
+        </div>
+      ))}
+
+    </div>
```
Comment (Muhammad0602) [inline src/components/reservation/Reservations.js]: You should also add how many hours the user reserved the appointment for. For example, you could say that from 4 pm to 6 pm and overall, for 2 hours. Then you could also show the overall price for the appointment.

**Review by Muhammad0602 — CHANGES_REQUESTED** [review]
Hi @mrkamin  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- The app works as expected ✔️ 
- Linter checks pass ✔️ 
- All the requirements completed ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Please review the comments below.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/home/Home.js (hunk 1 above)
Comment (mrkamin) [reply src/components/home/Home.js]: Hi @Muhammad0602,
thanks for your good and valuable feedback, your suggestions are applied.
Capture2

File: src/components/reservation/Reservations.js (hunk 2 above)
Comment (mrkamin) [reply src/components/reservation/Reservations.js]: Hi @Muhammad0602,
I think it is not in the requirements, 
Capture

**Review by Muhammad0602 — APPROVED** [review]
Hello @mrkamin .
Good job, the project is complete.

**Pull request merged**